.gitignore

```diff
@@ -46,6 +46,7 @@ packages/core/template/*/_site
 
 # Generated site (MarkBind)
 packages/cli/test/functional/*/_site
+packages/cli/test/functional/test_site/expected/diagrams/*.png
```

Comment:
We will still need to track the changes of the generated diagrams when there are updates to the test `.puml` files [here](https://github.com/MarkBind/markbind/tree/master/packages/cli/test/functional/test_site/diagrams) (add/remove `.puml` files or simply updating the code in the `.puml` files etc). 

What I meant earlier about "ignoring" the generated images is when there are no changes made to the original `.puml` test files before running `updatetest`. In this scenario, we should revert the changes made to the expected images generated after `updatetest` as the images should not have changed. Sorry for the confusion 😅 

Just need to remove this line here and the rest is good 👍